addons/survey/views/survey_templates.xml

```diff
@@ -597,5 +588,31 @@
             </div>
         </t>
     </template>
+
+    <template id="survey_navigation" name="Survey: Navigation">
```

Comment:
[C6] linting: I would move it to survey_templates_management.xml (in tools / utilities), to keep main templates only here (already a big file)